client/me/concierge/shared/appointment-info.js

```diff
@@ -146,9 +146,7 @@ class AppointmentInfo extends Component {
 				<Confirmation
 					title={ translate( 'Your upcoming appointment' ) }
 					description={ translate(
-						'We can talk about anything related to your site. ' +
-							'Get all your questions ready ' +
-							'-- we look forward to chatting!',
+						'Get all your questions ready ' + '-- we look forward to chatting!',
```

Comment:
While we're touching this, let's use a real mdash (—) character instead of the ASCII `--`.

Also, the `+` operator can be removed as the string is no longer that long and doesn't split over multiple lines.